nixos/modules/misc/nixpkgs/test.nix

```diff
@@ -0,0 +1,8 @@
+{ evalMinimalConfig, pkgs, lib, stdenv }:
+lib.recurseIntoAttrs {
+  invokeNixpkgsSimple =
+    (evalMinimalConfig ({ config, modulesPath, ... }: {
+      imports = [ (modulesPath + "/misc/nixpkgs.nix") ];
```

Comment:
```suggestion
      imports = [ (modulesPath + "/misc/nixpkgs.nix") (modulesPath + "/misc/meta.nix") ];
```

Reply:
this actually raises an interesting point: should modules include all modules they depend on? to be useful in a minimal eval that may be essential, but it'll also lead us into the same kind of include hell we have in C code (and possibly even slow down evals) :(

Reply:
> this actually raises an interesting point: should modules include all modules they depend on?

We could take the module system in this direction, possibly checking `config` for correctness in this regard. This would slow down per-module eval a bit, but is scalable, unlike the current approach. This would be quite a radical change, that I don't think we need to make.

Instead, I expect that we'll be able identify sets of essential modules at a few points in the spectrum from "things that can remotely call them selves systems" to regular NixOS. This will make determining the correct `imports` largely a non-issue. Ultimately, the answer is a yes. We should import what we depend on.

>  lead us into the same kind of include hell we have in C code

I don't expect it to, but even if it does, this was an experimental feature and it won't be noticed by regular NixOS users; probably not even many module authors.

Reply:
I've fixed it in the `nixpkgs.nix` module.

Reply:
that's better! :+1: